janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java

```diff
@@ -59,14 +60,14 @@ public JanusGraphVertexDeserializer(final JanusGraphHadoopSetup setup) {
         this.idManager = setup.getIDManager();
     }
 
+    private Boolean edgeExists(Vertex vertex, RelationCache possibleDuplicate) {
+        final RelationType type = typeManager.getExistingRelationType(possibleDuplicate.typeId);
```

Comment:
We already computed this `type` before, so no need to compute it again. Instead you could pass this `type` into this method. I.e. could be something like `private boolean edgeExists(Vertex vertex, RelationType type, RelationCache possibleDuplicate)`

Reply:
True, I'll change this as well.

Reply:
Thank you! I see that the latest force push didn't update the commit, so I unresolved this commits